.circleci/config.yml

```diff
@@ -186,7 +186,7 @@ workflows:
             branches:
               only: release
       - assume-role-staging:
-          context: api-assume-role-staging-context
+          context: api-assume-role-housing-staging-context
```

Comment:
I really wonder why this had chosen the wrong role to begin with. Did this come from an outdated baseapi?

Reply:
Actually as the search was one of the first ones we built, I think that the role name changed after we first set up the project.